src/geo/leaflet/leaflet-map-view.js

```diff
@@ -5,6 +5,10 @@ var MapView = require('../map-view');
 var LeafletLayerViewFactory = require('./leaflet-layer-view-factory');
 
 var LeafletMapView = MapView.extend({
+  initialize: function () {
+    MapView.prototype.initialize.apply(this, arguments);
+    this._linkMapHelpers();
```

Comment:
Since both subclasses (`leaflet-map-view` and `gmaps-map-view.js`) are invoking this ☝️, we can just move it up to the `initialize` method of the parent class (`map-view.js`)